### PR TITLE
clarification - android

### PR DESCRIPTION
**Pull request by abunur**
when exactly we should call  ‘ditto.auth.loginWithToken’

Thank you for your contribution to the Ditto documentation repository. 

--- 

### Description

_Add a description of your changes here_

### Checklist

- [ ] My code builds clean without any errors or warnings
- [ ] I have viewed your page in the netlify preview link (below) and confirmed that there are no broken links
- [ ] I have run my changes through a spell checker
- [ ] If I changed a link location, I added an entry to the `build/_redirects` file according to the [Netlify specification](https://docs.netlify.com/routing/redirects/redirect-options/)
- [ ] I have linked this PR to any existing issue by mentioning the issue in the description body
- [ ] I will close any relevant GitHub issues once my PR is merged

### CODE REVIEW

**Review by okdistribute — APPROVED** [review]

**Review by phatblat — APPROVED** [review]

File: docs/common/security/authentication.mdx [hunk 1]
```diff
@@ -403,6 +403,12 @@ authenticator->login_with_token("123", "my-auth", [](std::unique_ptr<DittoError>
 
 </TabItem>
 </Tabs>
+	
+When should you call `ditto.auth.loginWithToken`? 
+	
+The Ditto SDK itself will call `ditto.auth.loginWithToken`. So if you are using the callbacks described previously you should be covered, already. Ditto will try to opportunistically refresh whenever it can. However, if you know there are times when the app will be  online, you can also call it yourself explicitly if you want to.
```
Comment (contenuto) [inline docs/common/security/authentication.mdx]: If you are using the previously described callbacks, Ditto SDK automatically calls `ditto.auth.loginWithToken` to opportunistically refresh the access token; however, if you know that there are specific times that your app will have an internet connection, you can explicitly invoke the callback as well.

Comment (contenuto) [reply docs/common/security/authentication.mdx]: Is it "...that there are specific times that your app will have an internet connection, ..." or is it "...the specific times that your app will have an internet connection, ..."?

In other words, are we trying to explain that if the developer knows the specific times that the app has internet, they can trigger the callback to ensure that the access token gets refreshed? The window of opportunity, per se? If so, I suggest "...the specific times that your app will have an internet connection, you can explicitly invoke the callback during those times as well." Or something.

File: docs/common/security/authentication.mdx [hunk 2]
```diff
@@ -403,6 +403,12 @@ authenticator->login_with_token("123", "my-auth", [](std::unique_ptr<DittoError>
...
+
+There is also a branch on our Open Source Chat App demo called “authentication” that implements the authentication for Android as described in this tutorial. The repository can be found [here](https://github.com/getditto/demoapp-chat/tree/authentication).
```
Comment (contenuto) [inline docs/common/security/authentication.mdx]: `For an example implementation of authentication for Android, see the Ditto open-source demo chat app's  "[authentication](https://github.com/getditto/demoapp-chat/tree/authentication)" branch in the getditto GitHub repository.`

**Review by contenuto — COMMENTED** [review]
A few suggestions to consider.

**Pull request merged**